apps/right-to-rent-check/behaviours/tenancy-start-west-midlands-check.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const westMidPostcodes = require('../../../west-midland-postcodes.js');
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, callback) {
+    const tenancyStart = req.form.values['tenancy-start'];
+    const currentPostcode = req.sessionModel.get('property-address-postcode');
+    req.sessionModel.set('valid-tenancy', true);
+
+    if (tenancyStart < '2014-12-01') {
```

Comment:
Googling and I'm just trying to understand why do we need to replace `/` with `-`? as they seem to return the same results in both scenarios when I log the output